packages/cadl-ranch-specs/http/arrays/item-types/mockapi.ts

```diff
@@ -50,7 +65,9 @@ const Float32ValueMock = createModelMockApis("float32", [42.42]);
 Scenarios.Arrays_ItemTypes_Float32Value_get = passOnSuccess(Float32ValueMock.get);
 Scenarios.Arrays_ItemTypes_Float32Value_put = passOnSuccess(Float32ValueMock.put);
 
-const DatetimeValueMock = createModelMockApis("datetime", ["2022-08-26T18:38:00Z"]);
+const DatetimeValueMock = createModelMockApis("datetime", ["2022-08-26T18:38:00Z"], (datetime) =>
+  new Date(datetime).toISOString(),
```

Comment:
Hi @pshao25 , will this change payload sending in wire? what are the changes?

Reply:
No, it is a similar fix to https://github.com/Azure/cadl-ranch/blob/7d51edb53af6363b305562d02b728291abfaa46e/packages/cadl-ranch-specs/http/models/property-optional/mockapi.ts#L36-L39

Reply:
if it's a similar fix can we create a shared helper between these? it seems like we'll keep running into this issue